tests/review-pr.test.sh

```
@@ -1788,6 +1832,337 @@ else
+  printf '{"issue":0,"pr":73}\n' >"$dir/pr-context.json"
+  local extra
+  for extra in "$@"; do printf 'x\n' >"$dir/$extra"; done
+  if [ "$age" = stale ]; then
+    python3 -I -B -c 'import os,sys
+for path in sys.argv[1:]:
+    os.utime(path, (0, 0))' "$dir/locked" "$dir/pr-context.json"
+  fi
+}
+if (
+  set -u
+  RUN_ID=
```

Shellcheck (0.11.0)

[warning] RUN_ID appears unused. Verify use (or export if used externally).

(SC2034)